@uportal/notification-list/src/components/NotificationItem.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+    <article class="notification-item-container">
+        <div
+            class="media p-4 border-bottom d-flex justify-content-between"
+            :class="{ read: isRead, unread: !isRead, highlight }"
+        >
+            <img
+                class="media-image mr-3 rounded"
+                :src="notification.image"
+                :alt="notification.title"
+                v-if="notification.image"
+            />
+            <div
+                class="media-image mr-3 rounded img-placeholder"
+                :style="{ 'background-color': bgColor }"
+                v-else
+            />
+            <div class="media-body">
+                <h4 class="mt-0" v-if="itemHeading">
+                    {{ itemHeading }}
+                </h4>
+                <p>
+                    <strong v-if="notification.title">{{ notification.title }}:</strong>&nbsp;{{
+                        notification.body
+                    }}
+                </p>
+                <p class="media-link" v-if="notification.url">
+                    <a
+                        href="javascript:void(0)"
+                        target="_blank"
+                        @click="$emit('performaction', linkAction)"
+                    >
+                        {{ notification.linkText || notification.url }}
+                    </a>
+                </p>
+            </div>
+            <div
+                class="media-actions"
+                v-if="notification.availableActions && notification.availableActions.length"
+            >
+                <dropdown id="_uid" no-caret dropleft toggle-class="btn-icon">
+                    <template slot="button-content">
+                        <font-awesome-icon icon="ellipsis-v" size="2x" fixed-width />
+                    </template>
+                    <dropdown-item-button
+                        v-for="(action, num) in notification.availableActions"
+                        :key="num"
+                        @click="$emit('performaction', action)"
+                    >
+                        {{ action.label }}
+                    </dropdown-item-button>
+                </dropdown>
+            </div>
+        </div>
+    </article>
+</template>
+<script>
+import Dropdown from 'bootstrap-vue/es/components/dropdown/dropdown';
+import DropdownHeader from 'bootstrap-vue/es/components/dropdown/dropdown-header';
+import DropdownItemButton from 'bootstrap-vue/es/components/dropdown/dropdown-item-button';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons/faEllipsisV';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import { format } from 'date-fns';
+
+library.add(faEllipsisV);
+
+/*eslint-disable */
```

Comment:
Which eslint rules were causing issues?

Reply:
nope, was a holdover from trying to fix the optional property chaining. removed